src/RuleTest.php

```diff
@@ -44,10 +48,10 @@
 *   - actions
 *
 **/
-class PluginExampleRuleTest extends Rule {
+class RuleTest extends Rule {
 
    // From Rule
-   static public $right     = 'rule_import';
+   public $right     = 'rule_import';
```

Comment:
CommonDBTM defines this property as not static

Reply:
var renamed as submitted [here](https://github.com/pluginsGLPI/example/pull/38#discussion_r926381095) for RuleTestCollection